bin/GameIndex.dbf

```diff
@@ -35182,6 +35214,14 @@ Region = NTSC-J
 Serial = SLPS-73901
 Name   = Xenosaga Episode 1 - Der Wille zur Macht [PlayStation 2 The Best]
 Region = NTSC-J
+[patches = A3D63039]
```

Comment:
Side note (doesn't need to be addressed): I don't think this GameDB entry is actually accessible. Redump info shows that this disc is identical to SLPS-29002.